src/experiment.js

```diff
@@ -3,31 +3,49 @@ const path = require('path');
 const { FILEDIR } = require('./dirs');
 const { authMiddleware } = require('./authentication');
 
+const { timeStamp } = require('./serverUtils');
+
+var Config = require('./models/Config.model');
+
+const collection = 'experiment_data'
+
 module.exports = (router = new Router()) => {
-// Get config file
+  //get all config documents from mongo.
+  router.get('/', authMiddleware);
   router.get('/', (req, res) => {
-    res.sendFile(path.join(FILEDIR, 'config.txt'), {}, (err) => {
-      if (err) {
-        res.status(400).send({
-          message: 'no config',
-        });
-      }
+    Config.find({}).then(function(configs) {
+      var response = {};
+      var count = 1;
+      configs.forEach(function(record) { 
+        response[count] = JSON.parse(record["configData"]);
+        count += 1;
+      });
+      return Promise.all([response]);
+    }).then(function(respArray) {
+      res.status(200).set({
+      'Content-Type': 'application/json',
+      });
+      res.send(respArray[0]);
+    }).catch(function(error) {
+      res.status(500).send(error);
     });
   });
   // Post config requires password
   router.post('/', authMiddleware);
-  // Post config file
+  // Post config json object
   router.post('/', (req, res) => {
-    const { files: { file } = {} } = req;
-    if (!file) {
-      return res.status(400).send('Config file was not uploaded!');
+    const jsonData = req.body
+    if (Object.keys(jsonData).length === 0) {
+      return res.status(400).send('Malformed or bad data');
     }
-    // Save the config file
-    file.mv(path.join(FILEDIR, 'config.txt'), (err) => {
+    const time = timeStamp();
+    const config = new Config({time: time, configData: JSON.stringify(jsonData)});
+    config.save(function (err, fluffy) {
       if (err) {
+        console.log(err);
         return res.status(500).send(err);
       }
-      return res.send('Config saved');
```

Comment:
Would be better to send the configID number in a JSON response, so that the frontend admin panel can know which config it just created.

ex: 
```json
{
  "configID": 5
}